### DISCUSSION TITLE
Orion user authentication

### DISCUSSION THREAD

**Discussion by giacomochiarella** [PrefectHQ/prefect#6196] [Q&A]
Hi everyone! I've successfully installed prefect orion with an agent. Flows run smoothly. All of this is happening in an EC2 instance. I was wondering if there is a why to make Orion UI accessible through a password or something because, in this moment, anyone in the vpn would be able to access and play with flows.
I've enabled the access opening the port 4200 in the security group attached to the ec2 instance, which is accessible through the vpn.
Is there a way to secure Orion UI? Creating users or by just one password set in the server itself?

**Comment by NonlinearNimesh** [comment]
Hi, were you able to implement authentication for Orion UI, i wanted to put my orion server behine the authentication, For now i am using nginx proxy but it was not that secure. so i was exploring other ways i can implement it. Any help would be very helpful for us.
